puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
Rather than a separate shell builder step, you might need to use a list of files in *other* builders and add gemset_create.sh first.  (Bottom examples at http://docs.openstack.org/infra/jenkins-job-builder/definition.html#module-jenkins_jobs.local_yaml)